src/ralph/data_importer/management/commands/importer.py

```diff
@@ -111,21 +155,21 @@ def handle(self, *args, **options):
                             '',
                         ])
                         self.stdout.write(error_msg)
-            else:
-                if options.get('noinput'):
-                    result = model_resource.import_data(dataset, dry_run=False)
-                else:
-                    for idx, row in enumerate(result.rows):
-                        if not row.new_record:
-                            info_msg = 'Update: {} {}'.format(
-                                idx + 1,
-                                row.diff
-                            )
-                            self.stdout.write(info_msg)
-                    answer = input("type 'yes' to save data" + os.linesep)
-                    if answer == 'yes':
-                        result = model_resource.import_data(
-                            dataset,
-                            dry_run=False
-                        )
-                    self.stdout.write("Done")
+            if not options.get('noinput'):
+                self.stdout.write('Done')
+
+    def handle(self, *args, **options):
+        if not any([
+            options.get('dir'),
+            options.get('data_file'),
+            options.get('zipfile')
+        ]):
+            raise CommandError(
```

Comment:
since we only want to get one option, maybe there should be only one param with value (and 3 available choices), ex:
```
parser.add_argument(
    '-t', '--type',
    choices=['file', 'dir', 'zip'],
    required=True
)
```
?